.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
-* JabRef version (available in the About box):
-* Operating system and version:
 * Steps to reproduce:
   1. ...
   2. ...
   3. ...
-  * If applicable, excerpt of the bibliography file, screenshot, and excerpt of log (available in the error console)
+* If applicable, excerpt of the bibliography file, screenshot, and excerpt of log (available in the error console)
+* JabRef version (available in the About box):
```

Comment:
I liked to read the JabRef version first. Maybe we should reuse your idea from the PR template:

```
JabRef version <version as shown in the about box> on <Windows 10|Ubuntu 14.04|Mac OS X 10.8|...>
```